### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

File: source/guides/architecture/overview.md [hunk 1]
```diff
@@ -64,18 +64,20 @@ It's used as web delivery mechanism, for our features.
...
 ├── application.rb
+├── assets
+│   ├── favicon.ico
+│   ├── images
+│   ├── javascripts
+│   └── stylesheets
 ├── config
 │   └── routes.rb
 ├── controllers
-├── public
```
Comment (davydovanton) [inline source/guides/architecture/overview.md]: I think we should retain this folder as empty

Comment (mgrachev) [inline source/guides/architecture/overview.md]: This folder doesn't exist when you create a new project

Comment (AlfonsoUceda) [inline source/guides/architecture/overview.md]: @mgrachev yes, it's created

Comment (AlfonsoUceda) [inline source/guides/architecture/overview.md]: @mgrachev forget what I said, it exists in the root of the project haha :)

**Comment by AlfonsoUceda** [comment]
LGTM 👍

**Pull request merged**